### ISSUE TITLE
JSON serializer

### ISSUE THREAD

**Issue by jankotek** [jankotek/mapdb#735]
Started using ehcache for the requirement key is String and value com.google.gson.JsonObject. People does not like serialization and deserialization of huge JsonObject. I do not know is there any way in MapDB. even in ehcache for off-heap storage serialization and deserialization are required.

**Comment by debmalya** [comment]
Thanks a lot

**Comment by shabanovd** [comment]
@jankotek I have number of mapdb serializers for different json implementation. if you say what implementation required I can publish code. Or better to use new git repo for that?

**Comment by debmalya** [comment]
@shabanovd  this repository are you preparing for MapDB -json serializer https://github.com/shabanovd/mapdb-json ?

**Comment by shabanovd** [comment]
@debmalya yes, I did start to play locally to see what json options available for kotlin. There are no single standard library and that making huge headache. I'm ready to stop on any implementation, because universal serializer is not possible -(

**Comment by jankotek** [comment]
@shabanovd I have plan to support JSON serializers for major libraries. But right now there are many libraries. So using `mapdb-json` project with many optinal dependencies is probably best. 

Also there was some discussion to use Inflate compression with preset dictionary. That would work extremely well for  JSON or XML. See here:
https://groups.google.com/forum/#!topic/mapdb/0pszmoaGgK4

**Comment by samuelstein** [comment]
@jankotek do you know when the json serializer will be implemented?
